test/petstore.spec.ts

```diff
@@ -17,6 +17,6 @@ describe('updatePet', () => {
         const updatedPet: Pet = { name: 'FooPet', photoUrls: ['example.com'] }
         nock('http://www.example.com:80').put('/pet').reply(200, updatedPet)
 
-        expect(await client.updatePet({ body: updatedPet })).toEqual(updatedPet)
+        expect((await client.updatePet({ body: updatedPet })).right).toEqual(updatedPet)
```

Comment:
how come `right` is not triggering an error here? :thinking_face: there is no `isRight` guard around it